apps/appsets/components.yaml

```diff
@@ -205,6 +205,24 @@ spec:
                     - repoURL: '{{index .metadata.annotations "uc_deploy_git_url"}}'
                       targetRevision: '{{index .metadata.annotations "uc_deploy_ref"}}'
                       ref: deploy
+                - component: snmp-exporter
+                  componentNamespace: monitoring
+                  skipComponent: '{{has "snmp-exporter" ((default "[]" (index .metadata.annotations "uc_skip_components") | fromJson))}}'
+                  sources:
+                    - repoURL: '{{index .metadata.annotations "uc_repo_git_url"}}'
+                      targetRevision: '{{index .metadata.annotations "uc_repo_ref"}}'
+                      ref: understack
+                    - repoURL: '{{index .metadata.annotations "uc_deploy_git_url"}}'
+                      targetRevision: '{{index .metadata.annotations "uc_deploy_ref"}}'
+                      ref: deploy
+                    - repoURL: https://prometheus-community.github.io/helm-charts
+                      chart: prometheus-snmp-exporter
+                      targetRevision: 5.6.0
+                      helm:
+                        releaseName: prometheus-snmp-exporter
+                        valueFiles:
+                          - $deploy/{{.name}}/helm-configs/prometheus-snmp-exporter.yaml
```

Comment:
I'd ask that you flip this to the top under the `uc_repo` and `uc_deploy` as that's what we're using as our standard.

Reply:
https://rackerlabs.github.io/understack/deploy-guide/add-remove-app/